src/lib/accessibility/FocusZone/CHANGELOG.md

```diff
@@ -35,4 +35,16 @@ This is a list of changes made to this Stardust copy of FocusZone in comparison
     - `getParent` replaced with vanilla JS approach.
     - `getRTL` replaced with an `isRtl` property.
     - `createRef` replaced with a custom object and a callback which is necessary anyway because of custom component handling, see above for details.
-    - Focus related utilities moved to `focusUtilities.ts`.
\ No newline at end of file
+    - Focus related utilities moved to `focusUtilities.ts`.
+
+## FocusTrapZone Changelog
+
+This is a list of changes made to the Stardust copy of FocusTrapZone in comparison with the original [Fabric FocusTrapZone @ 0f567e05952c6b50c691df2fb72d100b5e525d9e](https://github.com/OfficeDev/office-ui-fabric-react/blob/0f567e05952c6b50c691df2fb72d100b5e525d9e/packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx).
+
+### feat(Accessibility): Add focus trap zone [#239](https://github.com/stardust-ui/react/pull/239)
+- using Stardust utils instead of Fabric utilities
+  - uses `eventStack`
+  - extends `React.Component` instead of Fabric `BaseComponent`
+  - uses `ReactDOM.findDOMNode` reference instead of `createRef` for `_root`
+- does not use `componentWillMount`
```

Comment:
nit: `componentWillMount` is not used anymore

Probably, this would be better, as it better reflects that this is a _change_ that was made.